csunplugged/topics/content/en/kidbots/programming-challenges/check-if-divisor/python-hints.md

```diff
@@ -0,0 +1,2 @@
+-   Divisor is a number that divides into another without a remainder (i.e. a number modulo its divisor is 0).
+    Use the `%` operator to calculate a modulo in Python. For example, `5 % 3` calculates 5 mod 3
```

Comment:
Due to the target age audience, possible change:
> calculates 5 mod 3

to

> calculates 5 modulo 3

Also missing full stop at end.